frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/pages/employee/GenerateOrders.js

```diff
@@ -1551,6 +1551,10 @@ const GenerateOrders = () => {
           warrantDetails: {
             issueDate: orderData?.auditDetails?.lastModifiedTime,
             caseFilingDate: caseDetails?.filingDate,
+            docType: orderDetails?.order?.additionalDetails?.formdata?.warrantType?.code,
```

Comment:
make common variable for `orderDetails?.order?.additionalDetails?.formdata` and use that variable